docs/application/native/guides/ui/efl/efl-util.md

```diff
@@ -375,6 +387,97 @@ To generate touch input events:
+          inputgen = efl_util_input_initialize_generator_with_name(EFL_UTIL_INPUT_DEVTYPE_TOUCHSCREEN, "Local_Device");
+
+          Since Tizen 5.0, you can create an input device synchronously with a given name (device_name can be NULL):
+          inputgen = efl_util_input_initialize_generator_with_sync(EFL_UTIL_INPUT_DEVTYPE_TOUCHSCREEN, NULL);
+       */
+       if (!inputgen) {
+           /* Failed to initialize the input generator system */
+
+           return;
+       }
+   ```
+
+2. After setting the input device type, to generate point input events call `efl_util_input_generate_pointer()`:
+
+   ```
+       ret = efl_util_input_generate_pointer(inputgen, 1, EFL_UTIL_INPUT_POINTER_MOVE, 100, 100);
+       if (ret != EFL_UTIL_ERROR_NONE) {
+           /* Failed to generate a pointer move event on (100, 100) (button is meaningless)*/
+           ret = efl_util_input_deinitialize_generator(inputgen);
+           if (ret != EFL_UTIL_ERROR_NONE)
+               /* Failed to deinitialize the input generator system */
+
+           return;
+       }
+       ret = efl_util_input_generate_pointer(inputgen, 1, EFL_UTIL_INPUT_POINTER_BUTTON_DOWN, 100, 100);
+       if (ret != EFL_UTIL_ERROR_NONE) {
+           /* Failed to generate a first button down event */
+           ret = efl_util_input_deinitialize_generator(inputgen);
+           if (ret != EFL_UTIL_ERROR_NONE)
+               /* Failed to deinitialize the input generator system */
+
+           return;
+       }
+       ret = efl_util_input_generate_pointer(inputgen, 1, EFL_UTIL_INPUT_POINTER_BUTTON_UP, 100, 100);
+       if (ret != EFL_UTIL_ERROR_NONE) {
+           /* Failed to generate a first button up event */
+           ret = efl_util_input_deinitialize_generator(inputgen);
+           if (ret != EFL_UTIL_ERROR_NONE)
+               /* Failed to deinitialize the input generator system */
+
+           return;
+       }
+
+       /* Since Tizen 5.5, you can generate wheel events: */
+       ret = efl_util_input_generate_wheel(inputgen, EFL_UTIL_INPUT_POINTER_WHEEL_VERT, 1);
+       if (ret != EFL_UTIL_ERROR_NONE) {
+           /* Failed to generate a vertical wheel event */
+           ret = efl_util_input_deinitialize_generator(inputgen);
+           if (ret != EFL_UTIL_ERROR_NONE)
+               /* Failed to deinitialize the input generator system */
+
+           return;
+       }
+
+       ret = efl_util_input_generate_wheel(inputgen, EFL_UTIL_INPUT_POINTER_WHEEL_HORZ, -1);
+       if (ret != EFL_UTIL_ERROR_NONE) {
+           /* Failed to generate a horizontal wheel event */
+           ret = efl_util_input_deinitialize_generator(inputgen);
+           if (ret != EFL_UTIL_ERROR_NONE)
+               /* Failed to deinitialize the input generator system */
+
+           return;
+       }
+   ```
+
+3. When no longer needed, free the `efl_util_inputgen_h` structure with the `efl_util_input_deinitialize_generator()` function:
```

Comment:
Please remove "the" before "`efl_util_input_deinitialize_generator()`" and the word "function". Apply this comment for this line only.

Rephrase to:
When no longer needed, free the `efl_util_inputgen_h` structure with `efl_util_input_deinitialize_generator()`: